bin/test

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+require 'kafo'
+require_relative '../hooks/boot/02-message-helpers'
+require 'ostruct'
+
+# Pretent we're a real Kafo
+Kafo::ColorScheme.new.setup
+kafo = OpenStruct.new(puppet_report: Kafo::PuppetReport.load_report_file('202309151500.yaml'))
```

Comment:
Commit this file to `test/fixtures` or put in `.gitignore` ? Note the PR has it committed right now. Could move this whole script into `test/` to avoid having to omit it in the packaging.